test-data/unit/pythoneval-asyncio.test

```diff
@@ -17,8 +17,7 @@ from typing import Any, Generator
 import asyncio
 from asyncio import Future
 
-@asyncio.coroutine
-def greet_every_two_seconds() -> 'Generator[Any, None, None]':
+async def greet_every_two_seconds() -> 'Generator[Any, None, None]':
```

Comment:
Can async defs still do `yield from`? I think you need `await` below.

Reply:
They cannot. Sorry, I should have marked this PR as a draft. I just did a find and replace and local tests have been finnicky and slow for a long time